src/sdk/accountspace_nb.js

```diff
@@ -217,39 +212,148 @@ class AccountSpaceNB {
     /////////////////////////////////
 
     async create_access_key(params, account_sdk) {
-        // TODO
-        dbg.log0('AccountSpaceNB.create_access_key:', params);
-        const { code, http_code, type } = IamError.NotImplemented;
-        throw new IamError({ code, message: 'NotImplemented', http_code, type });
+        const action = IAM_ACTIONS.CREATE_ACCESS_KEY;
+        const requesting_account = system_store.get_account_by_email(account_sdk.requesting_account.email);
+        const account_name = account_util.populate_username(params.username, requesting_account.name.unwrap());
```

Comment:
Could you rename this function so it would be clear that it returns account name?
Not in the scope of this PR.

Reply:
it's low priority and will pick it up if time permits.